recipe/meta.yaml

```
@@ -279,7 +273,9 @@ outputs:
         # other requirements
         - python
         - typing_extensions >=4.8.0
```

```suggestion
        - typing_extensions
```
here and elsewhere
https://github.com/pytorch/pytorch/blob/32f585d9346e316e554c8d9bf7548af9f62141fc/pyproject.toml#L10